xuname

```diff
@@ -8,8 +8,9 @@ OUTDATED=$(xbps-install -Mun)
 HOLD=$(xbps-query -H)
 VM=$(dmesg 2>/dev/null | awk '/Hypervisor detected/{print $NF}')
 [ -z "$VM" ] && grep -q hypervisor /proc/cpuinfo && VM=VM
-CPU=$(cat /proc/cpuinfo |awk '/^vendor_id/{print $NF;exit}')
-[ -z "$CPU" ] && CPU=$(cat /proc/cpuinfo |awk '/^platform/{print $NF;exit}')
+CPU=$(awk '/^vendor_id/{print $NF;exit}' /proc/cpuinfo)
+[ -z "$CPU" ] && CPU=$(awk '/^platform/{print $NF;exit}' /proc/cpuinfo)
+[ -z "$CPU" ] && CPU=$(lscpu | sed -n 's/^Vendor ID:[[:space:]]*//p')
```

Comment:
On my system `lscpu` speaks locale, not English, so `Vendor ID` matches nothing. `env LC_ALL=C lscpu`?